config/settings.yml

```diff
@@ -99,6 +99,7 @@
     :miq_namespace: management-infra
   :ems_amazon:
     :disabled_regions: []
+    :additional_regions:
```

Comment:
Either an empty array or empty Hash... either way I feel like the default should not be nil.